src/CircleGraph/CircleViewer.ts

```diff
@@ -123,9 +123,7 @@ export class CircleViewerProvider implements
   }
 
   // CustomReadonlyEditorProvider implements
-  async openCustomDocument(
-      uri: vscode.Uri, openContext: {backupId?: string},
-      _token: vscode.CancellationToken): Promise<CircleViewerDocument> {
+  async openCustomDocument(uri: vscode.Uri): Promise<CircleViewerDocument> {
```

Comment:
@seanshpark 

I understand that some of those parameters are for later usage. 
I suggested removing those unused-variables, but if you would like to leave them, let me know.
I will change this PR to put an underscore(_) to those unused parameters to let people know that those are not used and be ignored by the 'no-unused-variables' rule.

Reply:
I didn't know about the underscore and the tool you're going to use.
But please don't remove _token if possible.

Reply:
Sure :-D